app/views/downloads/download_linux.html.erb

```diff
@@ -12,49 +12,83 @@
 
   <h3>Debian/Ubuntu</h3>
   <p>For the latest stable version for your release of Debian/Ubuntu</p>
-  <code># apt-get install git</code>
+  <pre># apt-get install git</pre>
   <p>For Ubuntu, this PPA provides the latest stable upstream Git version</p>
-  <code># add-apt-repository ppa:git-core/ppa</code>
-  <code># apt update; apt install git</code>
+  <pre># add-apt-repository ppa:git-core/ppa</pre>
+  <pre># apt update; apt install git</pre>
+  <p>Optionally, if you would also like to install gitk, then run the following:</p>
+  <pre># sudo apt install gitk</pre>
   
   <h3>Fedora</h3>
-  <code># yum install git</code> (up to Fedora 21)<br>
-  <code># dnf install git</code> (Fedora 22 and later)
+  <pre># yum install git</pre> (up to Fedora 21)<br>
+  <pre># dnf install git</pre> (Fedora 22 and later)
+  <p>Optionally, if you would also like to install gitk, then run the following:</p>
+  <pre># sudo yum install tk</pre>
+  <pre># sudo yum install gitk</pre>
 
   <h3>Gentoo</h3>
-  <code># emerge --ask --verbose dev-vcs/git</code>
+  <pre># emerge --ask --verbose dev-vcs/git</pre>
+  <p>Check this <a href="https://bugs.gentoo.org/171230">Gentoo issue</a> for more information on how to install Gitk on Gentoo</p>
 
   <h3>Arch Linux</h3>
-  <code># pacman -S git</code>
+  <pre># pacman -S git</pre>
+  <p>Download Git and you will be prompted to download gitk. Check that optional dependency</p>
 
   <h3>openSUSE</h3>
-  <code># zypper install git</code>
+  <pre># zypper install git</pre>
+  <p>To install gitk:</p>
+
+  <h4>For standard run, execute the following:</h4>
+  <pre># zypper addrepo https://download.opensuse.org/repositories/openSUSE:Leap:15.2:Update/standard/openSUSE:Leap:15.2:Update.repo</pre>
+  <pre># zypper refresh</pre>
+  <pre># zypper install gitk</pre>
+  
+  <h4>For port run, execute the following:</h4>
+  <pre># zypper addrepo https://download.opensuse.org/repositories/openSUSE:Leap:15.2:Update/ports/openSUSE:Leap:15.2:Update.repo</pre>
+  <pre># zypper refresh</pre>
+  <pre># zypper install gitk</pre>
 
   <h3>Mageia</h3>
-  <code># urpmi git</code>
+  <pre># urpmi git</pre>
+  <p>If you would like to optionally install gitk: here is the <a href="https://madb.mageia.org/package/show/application/0/name/gitk">page for gitk</a> on Mageia's package catalog. It has a list of RPM files. Choose your RPM file and then run the following on it:</p>
+  <pre>$ rpm -i file.rpm</pre>
+  <p>Replace <i>file.rpm</i> with the path to the RPM file that you installed.</p>
 
   <h3>Nix/NixOS</h3>
-  <code># nix-env -i git</code>
+  <pre># nix-env -i git</pre>
+  <p>To install gitk:</p>
+  <pre># nix-env -iA nixos.gitAndTools.gitFull</pre>
 
   <h3>FreeBSD</h3>
-  <code># pkg install git</code>
+  <pre># pkg install git</pre>
+  <p>On FreeBSD gitk is installed with git.</p>
```

Comment:
I'd just as soon omit mentioning `gitk` if it already comes with `git`, in the name of brevity.

Reply:
I created a commit doing that.